src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -5555,23 +5570,37 @@ static void Main()
 
             var discard1 = GetDiscardDesignations(tree).First();
             Assert.Null(model.GetDeclaredSymbol(discard1));
+            Assert.True(model.GetSymbolInfo(discard1).IsEmpty);
```

Comment:
There seems to be a bug here. I'm investigating.

---
In reply to: [95025285](https://github.com/dotnet/roslyn/pull/16153#discussion_r95025285) [](ancestors = 95025285)